src/Kafka/CreateKafkaTopic/components/CreateTopicHead.tsx

```diff
@@ -0,0 +1,69 @@
+import type { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  PageSection,
+  PageSectionVariants,
+  Title,
+  Switch,
+  Breadcrumb,
+  BreadcrumbItem,
+} from "@patternfly/react-core";
+import { Link } from "react-router-dom";
+
+export type CreateTopicProps = {
+  showAllOptions: boolean;
+  kafkaName: string;
+  kafkaPageLink: string;
+  kafkaInstanceLink: string;
+  onShowAllOptions: (value: boolean) => void;
+};
+
+export const CreateTopicHead: VoidFunctionComponent<CreateTopicProps> = ({
+  showAllOptions,
+  onShowAllOptions,
+  kafkaName,
+  kafkaPageLink,
+  kafkaInstanceLink,
+}) => {
+  const { t } = useTranslation(["kafkaTemporaryFixMe"]);
```

Comment:
instead of kafkaTemporaryFixMe, you need to pass the locale json file which you need in this particular file

Reply:
Done